docs/remotes.md

```diff
@@ -0,0 +1,82 @@
+# Remotes
+
+Manage set of 3scale instances.
+
+Added remotes are stored in configuration file and can be used in any command
+where 3scale instances need to be specified.
+
+## Synopsis
+
+```
+3scale remote [--config-file <config_file>]
+3scale remote add [--config-file <config_file>] <name> <url>
+3scale remote remove [--config-file <config_file>] <name>
+3scale remote rename [--config-file <config_file>] <old_name> <new_name>
+```
+
+## Options
+
+*--config-file <config_file>*
+
+3scale CLI configuration file. When not set, the toolbox will lookup path at:
+
+* *THREESCALE_CLI_CONFIG* environment variable
+* `$HOME/.3scalerc.yaml`
+
+## Remote URLS
+
+The 3scale toolbox access 3scale instances by an `HTTP[S]` URL.
+Tokens are used for authentication and authorization purposes.
+3scale API supports the following token types:
+* `access_token` (preferred)
+* `provider_key`
+
+The following syntax is used:
+
+```
+http[s]://<provider_key>|<access_token>@<3scale-instance-domain>
+```
+
+## Commands
+
+### List
```

Comment:
It might be better to actually have a subcommand `list` and `3scale remote` just show a list of subcommands.

Reply:
`git remote` gives the list of remotes

Then you have `git remote add`, `git remote rename`, etc...

I think it kind of makes sense.

for list of subcommands, just 
```shell
$ 3scale remote -h
NAME
    remote - 3scale CLI remotes

USAGE
    3scale remote <command> [options]

DESCRIPTION
    3scale CLI command to manage your remotes

SUBCOMMANDS
    add        3scale CLI remote add
    remove     3scale CLI remote remove
    rename     3scale CLI remote rename

OPTIONS FOR 3SCALE
    -c --config-file=<value>      3scale CLI configuration file (default:
                                  /home/eguzki/.3scalerc.yaml)
    -h --help                     show help for this command
    -k --insecure                 Proceed and operate even for server
                                  connections otherwise considered insecure
    -v --version                  Prints the version of this command
```

Reply:
Git should not be called out as a well designed CLI.
I know what it does and I'm saying it might be better if the `3scale remote`  to print the list of subcommands available and use `3scale remote list` to list them, because of discoverability.

I'd say `kubectl` and `oc` are well designed CLI that give you at each step enough information.

Reply:
let me do this change in another PR